prelude/prelude.purs

```diff
@@ -61,6 +61,14 @@ module Prelude
   infixr 9 >>>
   infixr 9 <<<
 
+  -- | Semigroupoids are Categories, less `id`, which is to say they consist of objects and
```

Comment:
You should be able to link to `Category` and get links in the generated Markdown like this:

    A `Semigroupoid` is similar to a [`Category`](#Category) but lacks `id`...